packages/dev-server-rollup/test/node/plugins/node-resolve.test.ts

```diff
@@ -129,7 +129,8 @@ describe('@rollup/plugin-node-resolve', () => {
     }
   });
 
-  it('node modules resolved outside root directory are rewritten with commonjs', async () => {
+  // works with @rollup/plugin-commonjs 22.x but not with 21.x which we use to test?
```

Comment:
Can we update to latest commonjs plugin?

Reply:
if we do then some other tests fail 🙈

Reply:
e.g. then bare import to the commonjs helpers does not get rewritten... 😅

but v22 came out 10 hours ago... so maybe for next time

Reply:
Ok lets leave it then